src/Controller/AdminActivityController.php

```diff
@@ -84,6 +94,11 @@ public function new(Request $request): Response
             $entityManager->persist($activity);
             $entityManager->flush();
 
+            $this->addFlash(
+                'notice',
+                'L\'activité a bien été ajoutée.'
+            );
```

Comment:
Il  faut que tu fasses les messages sous ce format : 
`$this->addFlash(
'success',
'Le temps dédié pour chaque activité a bien été enregistré'
);`

Le 'success' correspond à la classe bootstrap qui va définir la couleur du message, ici du vert. Ca serait un message d'erreur, t'aurais mis 'danger'.